olx_parser.py

```diff
@@ -0,0 +1,124 @@
+import requests
+from bs4 import BeautifulSoup
+from datetime import datetime, timedelta
+
+
+date = {'январь': 'January',
```

Comment:
```suggestion
month_mapping = {'январь': 'January',
```
а ещё лучше будет получать значения из этого словаря через функцию
```python
def translate_month_to_en(ru_month_name: str) -> str:
    ...
```